From c6914bef9b867fb37ac7eb9e7752683f245db177 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Fri, 3 Aug 2018 10:19:51 +0800
Subject: [PATCH 20/54] media: intel-ipu4: css: scci_IPU4_master_20180802_0150
 fw release

Message for Open Source:
New firmware css release.

Message for Internal:
[Feature]New firmware css release.
[Root Cause/Changes]
FW_Manifest_ID: 251e7e439060428c83897c5241b78d6fdabac0de
FW_Build_Branch: IPU4_master
FW_Build_Topic: auto_merge_queue
library version: commit-id ddccabc
FW Bundle patch number of this release :  345978
FW tag : scci_IPU4_master_20180802_0150

Change-Id: Ib68332d6808573c4b1c2f23c49c4e2cb37518b1bb
Depends-on: Ie276c1489a9d83d6c93ce68f3e77a0e3247d485b
Depends-on: I6e470a2c6f64d6436fbffe08863beea4c088ddd1
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index a5da327..aea2a25 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180730
+#define IA_CSS_FW_PKG_RELEASE  0x20180802
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index a5da327..aea2a25 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180730
+#define IA_CSS_FW_PKG_RELEASE  0x20180802
-- 
1.9.1

